README.md

```diff
@@ -30,7 +30,7 @@
 
 ## About EddieBotUI
 
-EddieBotUI is a web application built to display all kinds of information about the **Eddie Jaoude Discord community** - [click to join](https://discord.gg/fPss8W9) - like users biography, usage stats, badges and more.
+EddieBotUI is a web application built to display all kinds of information about the **EddieHub Discord community** - [click to join](https://discord.com/invite/jZQs6Wu) - like users biography, usage stats, badges and more.
```

Comment:
How about we use the same invite link as the other repos (this is related to [this issue](https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/issues/163#issuecomment-826338069))? This way we don't need to change when the link is invalid 🙂 
```suggestion
EddieBotUI is a web application built to display all kinds of information about the **EddieHub Discord community** - [click to join](http://discord.eddiehub.org/) - like users biography, usage stats, badges and more.
```

Reply:
That's awesome. With this, we don't have to change the link again and again.
Thanks for helping me out  :-)

Reply:
@ShaileshKumar007 please merge this change? 🤔